<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1262</width>
    <height>664</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="signIn" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1200</width>
      <height>800</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <widget class="QLineEdit" name="password">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>300</y>
       <width>421</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="name">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>230</y>
       <width>421</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="userPassWord">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>300</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>  密码：</string>
     </property>
    </widget>
    <widget class="QLabel" name="userName">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>230</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>   用户名：</string>
     </property>
    </widget>
    <widget class="QPushButton" name="loginIn">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>400</y>
       <width>200</width>
       <height>100</height>
      </rect>
     </property>
     <property name="text">
      <string>登录</string>
     </property>
    </widget>
    <widget class="QPushButton" name="registe">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>400</y>
       <width>200</width>
       <height>100</height>
      </rect>
     </property>
     <property name="text">
      <string>注册</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>65</y>
       <width>611</width>
       <height>121</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:36pt;&quot;&gt;宠物小精灵&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="gameName">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>-10</y>
      <width>561</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:36pt;&quot;&gt;        宠物小精灵&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QWidget" name="users" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1200</width>
      <height>800</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <widget class="QLabel" name="userPetPic_1">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>200</width>
       <height>300</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_next_1">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>400</y>
       <width>100</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>下一个</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_name_1">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>0</y>
       <width>200</width>
       <height>50</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_hp_1">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>200</y>
       <width>200</width>
       <height>50</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_lv_1">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>100</y>
       <width>200</width>
       <height>50</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_kind_1">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>50</y>
       <width>200</width>
       <height>50</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_xp_1">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>150</y>
       <width>200</width>
       <height>50</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_atk_1">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>250</y>
       <width>200</width>
       <height>50</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_def_1">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>300</y>
       <width>200</width>
       <height>50</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_interval_1">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>350</y>
       <width>200</width>
       <height>50</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_lvUp">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>350</y>
       <width>100</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>升级</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_loginOut">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>650</y>
       <width>200</width>
       <height>100</height>
      </rect>
     </property>
     <property name="text">
      <string>退出登录</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>969</x>
       <y>100</y>
       <width>231</width>
       <height>600</height>
      </rect>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
     </property>
    </widget>
    <widget class="QLabel" name="userPetPic_2">
     <property name="geometry">
      <rect>
       <x>525</x>
       <y>0</y>
       <width>200</width>
       <height>300</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_name_2">
     <property name="geometry">
      <rect>
       <x>725</x>
       <y>0</y>
       <width>200</width>
       <height>50</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_kind_2">
     <property name="geometry">
      <rect>
       <x>725</x>
       <y>50</y>
       <width>200</width>
       <height>50</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_lv_2">
     <property name="geometry">
      <rect>
       <x>725</x>
       <y>100</y>
       <width>200</width>
       <height>50</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_xp_2">
     <property name="geometry">
      <rect>
       <x>725</x>
       <y>150</y>
       <width>200</width>
       <height>50</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_hp_2">
     <property name="geometry">
      <rect>
       <x>725</x>
       <y>200</y>
       <width>200</width>
       <height>50</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_atk_2">
     <property name="geometry">
      <rect>
       <x>725</x>
       <y>250</y>
       <width>200</width>
       <height>50</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_def_2">
     <property name="geometry">
      <rect>
       <x>725</x>
       <y>300</y>
       <width>200</width>
       <height>50</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_interval_2">
     <property name="geometry">
      <rect>
       <x>725</x>
       <y>350</y>
       <width>200</width>
       <height>50</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_next_2">
     <property name="geometry">
      <rect>
       <x>775</x>
       <y>400</y>
       <width>100</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>寻找对手</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_userName">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>600</y>
       <width>200</width>
       <height>50</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_lvUpBattle">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>350</y>
       <width>50</width>
       <height>100</height>
      </rect>
     </property>
     <property name="text">
      <string>升级赛</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_battle">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>350</y>
       <width>50</width>
       <height>100</height>
      </rect>
     </property>
     <property name="text">
      <string>决斗赛</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_nowPet">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>300</y>
       <width>150</width>
       <height>50</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>500</y>
       <width>500</width>
       <height>200</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_userName">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>550</y>
       <width>100</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;用户&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_battleInfo">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>450</y>
       <width>200</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;战斗信息：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_winRate">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>600</y>
       <width>100</width>
       <height>50</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_nowUser">
     <property name="geometry">
      <rect>
       <x>1000</x>
       <y>0</y>
       <width>200</width>
       <height>100</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;当前在线用户&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_jiangbei">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>550</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="throw_2" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1200</width>
      <height>800</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>65</y>
       <width>611</width>
       <height>121</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt;&quot;&gt;请选择一只宠物小精灵送出&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_throwName1">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>200</y>
       <width>100</width>
       <height>200</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_throwName2">
     <property name="geometry">
      <rect>
       <x>475</x>
       <y>200</y>
       <width>100</width>
       <height>200</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_throwName3">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>200</y>
       <width>100</width>
       <height>200</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_throwLv1">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>400</y>
       <width>100</width>
       <height>50</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_throwXp1">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>450</y>
       <width>100</width>
       <height>50</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_throwLv2">
     <property name="geometry">
      <rect>
       <x>475</x>
       <y>400</y>
       <width>100</width>
       <height>50</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_throwXp2">
     <property name="geometry">
      <rect>
       <x>475</x>
       <y>450</y>
       <width>100</width>
       <height>50</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_throwLv3">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>400</y>
       <width>100</width>
       <height>50</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_throwXp3">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>450</y>
       <width>100</width>
       <height>50</height>
      </rect>
     </property>
    </widget>
   </widget>
   <zorder>throw_2</zorder>
   <zorder>gameName</zorder>
   <zorder>users</zorder>
   <zorder>signIn</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1262</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
